ftw/addressblock/profiles/default/propertiestool.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0"?>
+<object name="portal_properties" meta_type="Plone Properties Tool">
+
+    <object name="navtree_properties" meta_type="Plone Property Sheet">
+        <property name="metaTypesNotToList" type="lines" purge="False">
+            <element value="ftw.addressblock.AddressBlock"/>
+        </property>
+    </object>
+
+    <object name="site_properties" meta_type="Plone Property Sheet">
+        <property name="types_not_searched" type="lines" purge="False">
```

Comment:
@maethu, is it correct to make the address block not searchable? The other blocks from ftw.simplelayout are not searchable either.